### PR TITLE
mkcloudconf: set correct ns for deployments (SOC-10288)

### PR DESCRIPTION
**Pull request by gosipyan**
Set correct ns for deployments

### CODE REVIEW

**Comment by gosipyan** [comment]
And its finally not failing  https://ci.suse.de/job/openstack-crowbar/394/

File: scripts/jenkins/cloud/ansible/roles/cloud_generator/templates/mkcloud.config.j2 [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {% set ns = namespace() %}
 {% include 'barclamps/lib/common.yml.j2' %}
+{% set deployments = ns.deployments %}
+{% include 'barclamps/lib/common.yml.j2' %}
```
Comment (stefannica) [inline scripts/jenkins/cloud/ansible/roles/cloud_generator/templates/mkcloud.config.j2]: You duplicated the line above

Comment (gosipyan) [reply scripts/jenkins/cloud/ansible/roles/cloud_generator/templates/mkcloud.config.j2]: fixed

**Comment by gosipyan** [comment]
testing https://ci.suse.de/job/openstack-crowbar/395/

**Comment by stefannica** [comment]
Fast-merging trivial fix.

**Pull request merged**